AsciiPixel/src/ascii_encode.jl

```diff
@@ -94,20 +94,19 @@ function encodeimg(
         end
         println(io, Crayon(reset = true))
     end
-    replace.(readlines(seek(io,0)), Ref("\n" => ""))::Vector{String}, h, 2w
+    replace.(readlines(io), Ref("\n" => ""))::Vector{String}, h, 2w
 end
 
-# colorant vector
-function encodeimg(
-        enc::SmallBlocks,
+function ascii_encode(
+        ::SmallBlocks,
         colordepth::TermColorDepth,
         img::AbstractVector{<:Colorant},
         maxwidth::Int = 80)
     maxwidth  = max(maxwidth, 5)
     while size(img, 1) > maxwidth
         img = restrict(img)
     end
```

Comment:
For me it might be a bad choice for encoding backend to downsample the image; what if we want to provide `ascii_decode`?

Reply:
Yes, I'm moving this upwards.